api/room_resource/schema.py

```diff
@@ -14,6 +14,67 @@ class Meta:
         model = ResourceModel
 
 
+class PaginatedResource(graphene.ObjectType):
+    pages = graphene.Int(pages=graphene.Int())
```

Comment:
When we pass a value to a field, it should be an expression that when executed sets the default value for the attribute, or just the default value for the attribute e.g False for Boolean fields, It would then add  expressions such as `pages=graphene.Int()` for pages,  has_next and has_previous attributes, consider removing the values passed to this attributes, remember we set the values in the resolvers